backend/app/models/investor.rb

```diff
@@ -10,7 +10,7 @@ class Investor < ApplicationRecord
   validates :impacts, array_inclusion: {in: Impact::TYPES}
   validates :sdgs, array_inclusion: {in: Sdg::TYPES}
   validates :investor_type, inclusion: {in: InvestorType::TYPES}
-  validates :language, inclusion: {in: I18n.available_locales.map(&:to_s)}
+  validates :language, inclusion: {in: (I18n.available_locales - [:zu]).map(&:to_s)}
```

Comment:
Maybe it would be worthy to make`I18n.available_locales - [:zu]` more dry so it does not repeat all the time here. It should be possible to use `StaticModel` approach for this --> `Language::TYPES`, where
```
class Language
  TYPES = I18n.available_locales - [:zu]).map(&:to_s)
end
```
then it is super easy to add some extra stuff to language later on